tools/src/postgen-64types.ts

```diff
@@ -0,0 +1,42 @@
+
+/**
+ * This file adds the "string" annotation to all numbers with 64 bits.
+ *
+ * Properties that are numbers with 64 bits are (always) converted into a string
+ * when serialized into JSON.
+ *
+ * @see https://developers.google.com/protocol-buffers/docs/proto3#json
+ *
+ * Note: This library strictly expects JSON serialization for CloudEvent data payloads.
+ *
+ * @example
+ * {
+ *   "generation": "1610564600595212"
+ * }
+ *
+ * To parse these values into an idiomatic value, i.e. an "int64" rather than "string", we modify
+ * the Golang fields for 64 bit types (int64, uint64, float64):
+ *
+ * Before:
+ *     Generation              *int64              `json:"generation,omitempty"`
+ * After:
+ *     Generation              *int64              `json:"generation,string,omitempty"`
+ *
+ * This modification currently cannot be done at the JSON schema + Quicktype level,
+ * as JSON schema does not have a representation for 64 bit numbers serialized as strings.
```

Comment:
nit: It's great to see the context provided by this comment. However, since it implies that if JSON-Schema did gain such support we'd want to change this implementation something to mark how to track the need for that change would help.

* Good: Add https://github.com/json-schema-org/json-schema-spec/issues/361 to the comment block.
* Better: File an issue in this repo providing context on the decision made so far, the issue to watch for changes, and add a TODO on the local issue to the comment block.

Reply:
- Added GitHub issue.
- I am not going to file a local issue in this case, as it requires a specification change that isn't actionable in this repo. We can discuss more about this offine. Maybe we can file an issue in the google-cloudevents repo.